src/main/java/com/applitools/imagetester/lib/EyesFactory.java

```diff
@@ -120,7 +120,14 @@ public EyesFactory baselineEnvName(String envName) {
     }
 
     public EyesFactory logFile(String filename) {
-        this.logFilename = filename;
+    	this.logger.printMessage("------------------------------------------ \n"
+    			+ "Setting the log file using the 'lf' flag has been DEPRECATED. \n"
+    			+ "The log are automatically created and saved to: \n"
+    			+ "Mac/Linux - $TMPDIR/applitools-logs/ \n"
+    			+ "Windows - $env:TEMP/applitools-logs/ \n"
+    			+ "The log directory can now be specified using this environment variable: \n"
+    			+ "APPLITOOLS_LOG_DIR=<path> \n"
+    			+ "------------------------------------------ \n");
         return this;
     }
 
```

Comment:
I'm making a couple of small changes before I push up a release today.

Thanks for this!